src/PlaywrightSharp.Firefox/Helper/NetworkSetAuthCredentialsRequestJsonConverter.cs

```diff
@@ -0,0 +1,44 @@
+using System;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using PlaywrightSharp.Firefox.Protocol.Network;
+using PlaywrightSharp.Firefox.Protocol.Runtime;
+using PlaywrightSharp.Helpers;
+
+namespace PlaywrightSharp.Firefox.Helper
+{
+    /// <summary>
+    /// We need to be able to send null values in order to reset the username/password values.
+    /// </summary>
+    internal class NetworkSetAuthCredentialsRequestJsonConverter : JsonConverter<NetworkSetAuthCredentialsRequest>
```

Comment:
My concern here is whether we are going to need more of these or not. I wouldn't like to have 5 classes like this.

Reply:
I think this is the last one